app/src/main/res/values/dimens.xml

```diff
@@ -10,9 +11,17 @@
     <dimen name="layout_padding_16dp">16dp</dimen>
     <dimen name="layout_padding_24dp">24dp</dimen>
     <dimen name="layout_padding_30dp">30dp</dimen>
+    <dimen name="layout_padding_35dp">35dp</dimen>
+    <dimen name="layout_padding_40dp">40dp</dimen>
     <dimen name="layout_padding_50dp">50dp</dimen>
+    <dimen name="layout_padding_54dp">54dp</dimen>
+    <dimen name="layout_padding_60dp">60dp</dimen>
     <dimen name="layout_padding_64dp">64dp</dimen>
     <dimen name="layout_padding_75dp">75dp</dimen>
+    <dimen name="layout_padding_180dp">180dp</dimen>
```

Comment:
Try to reuse existing padding dimens or try using ConstraintLayout or RelativeLayout.

Reply:
hey, I have reused the existing dimensions wherever i can, and I feel that the ones that i have added are necessary to give a clean design.